modules/activity_tracker/variables.tf

```diff
@@ -105,13 +105,14 @@ variable "cos_bucket_endpoint" {
 # COS Targets
 variable "cos_targets" {
   type = list(object({
-    endpoint                   = string
-    bucket_name                = string
-    instance_id                = string
-    api_key                    = optional(string)
-    service_to_service_enabled = optional(bool, false)
-    target_region              = optional(string)
-    target_name                = string
+    endpoint                          = string
+    bucket_name                       = string
+    instance_id                       = string
+    api_key                           = optional(string)
+    service_to_service_enabled        = optional(bool, false)
+    target_region                     = optional(string)
+    target_name                       = string
+    skip_atracker_cos_iam_auth_policy = optional(bool, true)
```

Comment:
oh and this should be false too since we are defaulting `service_to_service_enabled` to true